### PR TITLE
fix(logger): simpler logger, fixed imports

### PR DESCRIPTION
**Pull request by mkucharz**

### CODE REVIEW

File: src/logger.js [hunk 1]
```diff
@@ -85,10 +76,10 @@ class Logger {
   }
 
   _parseArg(arg) {
-    const isObject = arg instanceof Object && arg !== null
+    const isObject = typeof (arg) === 'object'
```
Comment (Idered) [inline src/logger.js]:
```js
typeof null === 'object' // returns true
```

Do we want to handle null as object?

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+export {
+  users,
+  account,
+  instance,
+  event,
+  socket,
+  logger,
+  response,
+  data,
+  server
```
Comment (Idered) [inline src/index.js]: We probably should export server as default:

```js
export default server
```

Comment (Idered) [inline src/index.js]: We probably should export server as default:

```
export default server
```

File: package.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "test:e2e": "cross-env NODE_ENV=test nyc yarn run e2e",
     "test:coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
     "lint": "xo src/**/*.js test/**/*.js",
+    "lint:fix": "xo --fix src/**/*.js test/**/*.js",
```
Comment (Idered) [inline package.json]: Great. More scripts.

File: src/index.js [hunk 4]
```diff
@@ -1,3 +1,24 @@
 import server from './server'
```
Comment (Idered) [inline src/index.js]:
```js
import Server from './server'
```

File: src/index.js [hunk 5]
```diff
@@ -1,3 +1,24 @@
...
 
-export default server
+const initializedServer = server()
```
Comment (Idered) [inline src/index.js]: const server = new Server()

File: src/logger.js [hunk 6]
```diff
@@ -37,39 +32,35 @@ class Logger {
...
-    const time = chalk.gray(this._getNowString(now).split(' ')[1])
+    const diff = `+${this._calculateDiff(this._start, now)}`
+    const time = this._getNowString(now).split(' ')[1]
 
     if (!this._shouldLog(this._scope)) {
       return
     }
 
     // Level
     const levelName = this._pad(5, `${this._level}`, ' ')
```
Comment (Idered) [inline src/logger.js]: Just name it level.

File: src/logger.js [hunk 7]
```diff
@@ -37,39 +32,35 @@ class Logger {
...
-    const level = color ? chalk[color](levelName) : levelName
-
     args = args.map(this._parseArg).join(' ')
 
-    console.log(level, this._scope, time, diff, args)
+    console.log(levelName + ':', time, this._scope, args, diff, 'ms')
```
Comment (Idered) [inline src/logger.js]: Use template strings.

File: tests/e2e/instance.js [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import {expect} from 'chai'
 
-import server from '../../src'
+import {server} from '../../src'
```
Comment (Idered) [inline tests/e2e/instance.js]: Should be default import.

File: src/logger.js (hunk 1 above)
Comment (mkucharz) [inline src/logger.js]: f*cking js

File: src/index.js (hunk 2 above)
Comment (mkucharz) [inline src/index.js]: Feel free to try to do it :)

Comment (mkucharz) [inline src/index.js]: Feel free to try to do it :)

File: tests/e2e/instance.js (hunk 8 above)
Comment (mkucharz) [inline tests/e2e/instance.js]: Feel free to try to do it :)

**Pull request merged**